shark_turbine/kernel/wave/expansion.py

```diff
@@ -108,6 +108,25 @@ def is_contiguous_dim(
     return is_innermost_dim or all_unit_dims
 
 
+def compute_stride(
+    symbolic_shape: tuple[IndexSymbol, ...],
+    vector_shapes: dict[IndexSymbol, int],
+    target_dim: IndexSymbol,
+) -> int:
```

Comment:
Is return type always intended to be resolved to `int` at this point?

Reply:
Good question. Basically what you are asking is whether vector_shapes is always going to be specified as an int? Since it is used to represent the minimum granularity of the operations, I think it should always be specified as an int.

Reply:
We can do `return int(stride)` to force it (or get error).

Reply:
I think error is better since because forcing things to be int may not be unexpected.

Reply:
Okay will add a check here and throw an error if not an int.